index.html

```diff
@@ -5,16 +5,71 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="">
-    <meta name="author" content="">
+    <meta name="shopping cart" content="">
+    <meta name="Lentz" content="">
```

Comment:
hint hint: Look into the meta tag docs. This is how it should be <meta name="author" content="Lentz">